### DISCUSSION TITLE
Logo NOT added! || @svgWithLogo.php & @svgWithLogoAndCustomShapes.php

### DISCUSSION THREAD

**Discussion by X-Nightmare-X** [chillerlan/php-qrcode#288] [Q&A] [answered]
Hey guys, i am testing the lib and i got it running within a website where qr-interactive works and the settings on the site and the other stuff. 
Now i want to "brand" my QR Codes.
But i am getting problems with running the QR generator with adding logos.
Both examples mentioned in the title are running and generate QR Codes but without a logo added in the middle.
I didnt change much from the original examples. 
Testet with "original" github.svg and with my own .svg but it is not working.

Where is the failure, what am I doing wrong?

I want to add the QR Logo within the site with the qr-interactive script, so that i can click "branded" or not like the other settings on the html page.

**Comment by codemasher** [comment]
Hi, are you using the examples that match your installed library version? See https://github.com/chillerlan/php-qrcode?tab=readme-ov-file#documentation

**Reply by X-Nightmare-X** [reply]
Hey, have to check that.
But i installed the latest Version and copied the examples out of GH.
I think i will check the whole installation and doing it again only with composer and not with copying the source and then install. Evtl. that will work better. But it seems strange that all is working except the added Logo.

**Reply by codemasher** [reply]
That's the whole point - the examples are not necessarily interchangeable between major versions.

**Reply by X-Nightmare-X** [reply]
Ahhh okey xD

Do you know then if the both examples mentioned above are able to run with v5.0.2. ?
I installed it again and  I am testing it in a few minutes.

If they are not running with v5.0.2 what do i have to change?
Thanks for answering so fast.

**Reply by X-Nightmare-X** [reply]
Ok, I dont know what to do :/
Its working but without the logo again.
I am not that good to understand, what to change, that the logo will be inserted in the generated QR.

That would be so cool, if that would work.

The library/project is so cool at all. But that would be extra nice, if i could brand my QRs.

I am very thankful if you would help me getting it to work. And if I see my failure or what i have to change then i could evtl. learn what i have to do in the future.

**Reply by codemasher** [reply, marked as answer]
Can you please describe what exactly is the issue and maybe post an code sample of what you're trying to achieve? btw these are the examples for v5.x https://github.com/chillerlan/php-qrcode/tree/v5.0.x/examples

**Reply by X-Nightmare-X** [reply]
JEAH NICE :D
Thanks for the link.
That worked directly.
I only wanted to get a QR code with a logo inside.
The version u send me, worked with the v5.0.2. 

Sorry, i didnt know, that there are different version trees.
I am not so familiar with GitHub and finding that.
Sorry for taking your time, for that easy solution!

BIG THANKS to you!!

**Reply by X-Nightmare-X** [reply]
The only thing, is that now I have to get my added logo not so big. Its out of the box (like too big) for the box. How does it size automatic? With Github.svg it worked. With mine not. I can only see the top left corner of the svg

**Reply by codemasher** [reply]
Here's a bit about how to properly scale SVG logos: https://github.com/chillerlan/php-qrcode/discussions/268#discussioncomment-11051619

**Reply by codemasher** [reply]
Wait no here's the full thread including explanations https://github.com/chillerlan/php-qrcode/discussions/286
